File: week-02/day-01/seconds_in_a_day.py
current_hours = 14;
current_minutes = 34;
current_seconds = 42;

# Write a program that prints the remaining seconds (as an integer) from a
# day if the current time is represented bt the variables

hour = 3600
minute = 60
second = 1
day = 86400

current_hours = hour * current_hours
current_minutes = minute * current_minutes
current_seconds = second * current_seconds

a = current_hours + current_minutes + current_seconds
print(day - a)

File: week-07/day-01/seconds-in-a-day.js
'use strict';

var currentHours = 14;
var currentMinutes = 34;
var currentSeconds = 42;

// Write a program that prints the remaining seconds (as an integer) from a
// day if the current time is represented by these variables

var hour = 3600;
var minute = 60;
var second = 1;
var day = 86400;

currentHours = hour * currentHours;
currentMinutes = minute * currentMinutes;
currentSeconds = second * currentSeconds;

var a = currentHours + currentMinutes + currentSeconds;
console.log(day - a);